data/sdss/Makefile

```diff
@@ -13,17 +10,17 @@ FIELD4 := $(shell printf '%04d' $(FIELD_STRIPPED))
 SUBDIR2 := $(RUN_STRIPPED)/$(CAMCOL)
 SUBDIR3 := $(SUBDIR2)/$(FIELD_STRIPPED)
 
-all : $(SUBDIR3)/fpM-$(RUN6)-u$(CAMCOL)-$(FIELD4).fits \
-      $(SUBDIR3)/fpM-$(RUN6)-g$(CAMCOL)-$(FIELD4).fits \
-      $(SUBDIR3)/fpM-$(RUN6)-r$(CAMCOL)-$(FIELD4).fits \
-      $(SUBDIR3)/fpM-$(RUN6)-i$(CAMCOL)-$(FIELD4).fits \
-      $(SUBDIR3)/fpM-$(RUN6)-z$(CAMCOL)-$(FIELD4).fits \
+all : $(SUBDIR3)/fpM-$(RUN6)-u$(CAMCOL)-$(FIELD4).fit \
+      $(SUBDIR3)/fpM-$(RUN6)-g$(CAMCOL)-$(FIELD4).fit \
+      $(SUBDIR3)/fpM-$(RUN6)-r$(CAMCOL)-$(FIELD4).fit \
+      $(SUBDIR3)/fpM-$(RUN6)-i$(CAMCOL)-$(FIELD4).fit \
+      $(SUBDIR3)/fpM-$(RUN6)-z$(CAMCOL)-$(FIELD4).fit \
```

Comment:
I think @ismael-mendoza wanted to rename the `.fit` files to `.fits`

Reply:
Yes that would be great